provisioners/deploy-backend.yml

```
@@ -81,6 +87,7 @@
         - process-daemon.service
         - sqs-daemon.service
         - node-upload.service
+        - notification-service
```

https://github.com/PermanentOrg/infrastructure/blob/30b58ca34d740e7cee289927cd71b09d63ca06e2/provisioners/configure.sh#L70-L71
This needs to match the service definition file created in `configure.sh`:
```suggestion
        - notification.service
```